src/explore-education-statistics-frontend/src/middleware/chain.ts

```diff
@@ -0,0 +1,26 @@
+import { NextMiddlewareResult } from 'next/dist/server/web/types';
+import {
+  NextFetchEvent,
+  NextMiddleware,
+  NextRequest,
+  NextResponse,
+} from 'next/server';
+
+export type ChainedMiddleware = (
+  request: NextRequest,
+  event: NextFetchEvent,
+  next: NextMiddleware,
+) => NextMiddlewareResult | Promise<NextMiddlewareResult>;
+
+export default function chain(
```

Comment:
We should move this into `middleware/pages` as it's only relevant for page middleware